subsys/bluetooth/host/smp.c

```diff
@@ -1587,6 +1587,16 @@ int bt_smp_br_send_pairing_req(struct bt_conn *conn)
 	struct net_buf *req_buf;
 	uint8_t max_key_size;
 	struct bt_smp_br *smp;
+	uint8_t remote_fixed_chan;
+
+	/*
+	 * Please NOTE: The remote_fixed_chan type should be aligned with type
+	 * of returned value of function bt_l2cap_br_get_remote_fixed_chan.
+	 */
```

Comment:
I don't think this adds much value. The type on the left side of an assignment operation should ideally always match the type on the right side. I.e. just drop this code comment.

Reply:
Done.